### PR TITLE
LogicApp IntegrationAccount PSCmdlets

### PR DESCRIPTION
**Pull request by pankajsn**
LogicApp IntegrationAccount PSCmdlets

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@pankajsn__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: src/ResourceManager/LogicApp/Commands.LogicApp.Test/ScenarioTests/Common.ps1 [hunk 1]
```diff
@@ -73,6 +73,17 @@ function TestSetup-CreateResourceGroup
 
 <#
 .SYNOPSIS
+Creates named resource group to use in tests
+#>
+function TestSetup-CreateNamedResourceGroup([string]$resourceGroupName)
+{    
+    $resourceGroup = New-AzureRmResourceGroup -Name $resourceGroupName -location "brazilsouth" -Force
```
Comment (markcowl) [inline src/ResourceManager/LogicApp/Commands.LogicApp.Test/ScenarioTests/Common.ps1]: Please do not hard code locations.  use the Get-location or get-provider cmdlets to determine an available location (it's fine to prefer brazilsouth if available.  This makes it easier to maintain the tests, and to run them against different environments

**Comment by markcowl** [comment]
@pankajsn This PR is too big to review in Github.  In the future, please try to split these into smaller chunks.  To set expectations, will have to sync to your repo and diff locally, so this will likely not be reviewed until tomorrow.

**Comment by markcowl** [comment]
@pankajsn is this intended for AzureStack?  If so, please target the release-1.2.4 branch

**Comment by markcowl** [comment]
@pankajsn The PR has merge conflicts, please update.

**Comment by pankajsn** [comment]
Looks like there was an merge conflict in ChangeLog.md file which is updated by some recent pull request.

This conflict has been resolved now (See: http://azuresdkci.cloudapp.net/job/powershell/5791/ ). Please merge this PR.

Regards,
~Pankaj

**Comment by hovsepm** [comment]
@pankajsn please implement https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d#what-is-the-change

**Comment by hovsepm** [comment]
@pankajsn please target this pr to release-1.6.0 branch

**Comment by pankajsn** [comment]
#PR2567<https://github.com/Azure/azure-powershell/pull/2567> is created.

**Pull request closed without merging**